packages/mobile/src/import/saga.ts

```diff
@@ -160,8 +161,13 @@ export function* importBackupPhraseSaga({ phrase, useEmptyWallet }: ImportBackup
       ValoraAnalytics.track(AppEvents.redux_store_recovery_success, { account })
     }
     ValoraAnalytics.track(OnboardingEvents.wallet_import_success)
-
-    navigateClearingStack(Screens.VerificationEducationScreen)
+    const skipVerification = yield select(skipVerificationSelector)
+    if (skipVerification) {
+      yield put(initializeAccount())
```

Comment:
seems like there's probably a good answer to this, but can you explain to me why we only need to initialize the account if we're skipping verification?

Reply:
Currently `initializeAccount` is only called on the VerificationEducationScreen OR if it gets skipped because komenci is down or verification is disabled completely. I'm not entirely sure why those were the places chosen, but I though it probably best to continue calling `initializeAccount` when verification is skipped